appinventor/blocklyeditor/src/replmgr.js

```diff
@@ -334,6 +334,21 @@ Blockly.ReplMgr.putYail = (function() {
             var offer;
             var key = rs.replcode;
             var haveoffer = false;
+            var connectionstate = "none";
+            var webrtcerror = function(doalert, msg) {
+              webrtcdata = null;
+              webrtcstarting = false;
+              webrtcrunning = false;
+              webrtcforcestop = true;
+              top.BlocklyPanel_indicateDisconnect();
+              top.ConnectProgressBar_hide();
+              if (!webrtcpeer) {
+                webrtcpeer.close();
+              }
+              if (doalert) {
+                window.alert(msg);
```

Comment:
I'd prefer if we use Blockly.Util.Dialog rather than window.alert (ref #1039)

Reply:
Oh, OK :-)

Reply:
Done